### PR TITLE
Add communication methods [#17]

### PR DESCRIPTION
**Pull request by levidurfee**
Please let me know your thoughts, or if anyone has any suggestions.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,3 +6,9 @@ The AMP Advisory Committee (AC) provides perspective and advice to the Technical
 - [Our working mode](https://github.com/ampproject/meta-ac/blob/master/WORKING_MODE.md)
 - [Meeting minutes](https://github.com/ampproject/meta-ac/tree/master/meetings)
 - [Upcoming meetings](https://github.com/ampproject/meta-ac/labels/AC%20Meeting)
+
+## Communication Channels
+
+- [Slack](https://docs.google.com/forms/d/e/1FAIpQLSd83J2IZA6cdR6jPwABGsJE8YL4pkypAbKMGgUZZriU7Qu6Tg/viewform?fbzx=4406980310789882877) [#amp-ac](https://amphtml.slack.com/messages/amp-ac/) - Public
```
Comment (tobie) [inline README.md]: Can you make the link to the form a bit more explicit? e.g. "To get invited to the Slack workspace, add your email to this form."

File: README.md [hunk 2]
```diff
@@ -6,3 +6,9 @@ The AMP Advisory Committee (AC) provides perspective and advice to the Technical
...
+- Video Conferencing - Private
```
Comment (tobie) [inline README.md]: The archives are private, but joining the video conf isn't. Can we define how to do it? Maybe adding the URL to join the zoom chat on the agenda? And can we reference that here?

File: README.md [hunk 3]
```diff
@@ -6,3 +6,9 @@ The AMP Advisory Committee (AC) provides perspective and advice to the Technical
...
+- Email Group - Private
```
Comment (tobie) [inline README.md]: We should limit the conversation on the mailing list to logistical and/or matters and say so here.

File: README.md (hunk 1 above)
Comment (levidurfee) [reply README.md]: Addressed in 7987f08

File: README.md (hunk 2 above)
Comment (levidurfee) [reply README.md]: Addressed in 86d11ab

File: README.md (hunk 3 above)
Comment (levidurfee) [reply README.md]: Addressed in b98fc89

**Pull request merged**